docs/releases/readme.md

```diff
@@ -45,6 +45,10 @@ _Outcome_: **Team is aware of release and in agreement about what fixes & featur
     -   For _major_ and _minor_ releases, create branch: `release/X.X`.
     -   For _patch_ releases, the branch should already exist.
 
+#### Create a release pull request
+
+Using the [release pull request template](../../.github/pull_request_template.md), create a pull request for the release branch you just created. This pull request will *not* get merged to master but contains the checklist to go over as a part of the release along with being a place to contain all planning/communication around the release. The checklist should be completed and the pull request has an approved review from at least one team member before you do the Github deploy or release the plugin to WordPress.org.
```

Comment:
For me the weirdest thing about this process was using Pull Requests for something which is not merged. Have you considered using the PR template request to merge in the testing instructions (docs/testing/releases/x) so once the PR is approved and the release happens, the PR is merged rather than closed?

Reply:
Ahh yeah that is interesting/fair point. 

>  Have you considered using the PR template request to merge 

Could do [version bumps](https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/5e92176e9840d93f5c440f09fefb3e6f24758c06#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) with the release I guess (still getting used to the workflow) - or is that somewhat automated?

---

Using something like gitflow you would have a scenario where the the develop/release branch would be merged into master (or similar).

Reply:
I agree it's a bit weird, but I'm also thinking through the lense of eventual automation.  Still, thinking on it, we _should_ be merging because things like:

- {$_VID_} replacement for `@since` tags
- Changelog
- readme.txt changes
- docs around tests etc (that you noted)

Should get merged back. I'm going to revisit this and we might need to change a bit of the process on where version replacement is done and committed so that we _can_ merge back to master.

Reply:
So generally speaking, we do need to make sure changes get merged back to master (`$:VID:` tag replacement, changelog, docs updates etc) but where things get tricky is when there are cherry-picks in a release branch (which happened for both `2.6.0` and `2.6.1`. So I don't think we can do an automated straight merge to master because of the conflicts from the cherry-picks (unless the automation does the merge recursively favoring a branch - which is potentially dangerous). Maybe what the automation could do is:

- attempt to do the merge of the pull request.
- if that fails then add a comment to the pull request outlining the conflicts and the release lead would have to manually merge.

But generally speaking we should _favor_ merging pull request changes back into master so I'll update the documentation to include that.

Reply:
I just did a direct commit to master to bring in the `$:VID:` replacement changes done for the 2.6.0 release (see bbe27f7e3aff)